inst/extdata/reportingVariables/Mapping_EuropeanEnergyDatasheets.csv

```diff
@@ -242,9 +258,10 @@
 "Energy Balance|Final energy consumption|Others";"Mtoe";"FE|Other Sector (EJ/yr)";0.041868
 "Electricity Production|Gross Electricity Generation";"TWh";"SE|Electricity (EJ/yr)";0.0036
 "Electricity Production|Solid fossil fuels, peat and products, oil shale and oil sands";"TWh";"SE|Electricity|Solids (EJ/yr)";0.0036
```

Comment:
can you remove this line?
("Electricity Production|Solid fossil fuels, peat and products, oil shale and oil sands";"TWh";"SE|Electricity|Solids (EJ/yr)";0.0036)